### ISSUE TITLE
duplicate recipes

### ISSUE THREAD

**Issue by bell07** [minetest-mods/mydoors#6]
I  am writing a small QA-check-mod and my script found the next items with identical recipes:

```
same recipe my_future_doors:door6a 1    my_future_doors:door2a 1
same recipe ts_doors:trapdoor_default_steelblock    doors:trapdoor_steel
same recipe my_future_doors:door6a 1    my_future_doors:door9a 1
same recipe my_future_doors:door2a 1    my_future_doors:door9a 1
same recipe my_future_doors:door6a 1    my_future_doors:door8a 1
same recipe my_future_doors:door2a 1    my_future_doors:door8a 1
same recipe my_future_doors:door9a 1    my_future_doors:door8a 1
same recipe my_future_doors:door6a 1    my_future_doors:door7a 1
same recipe my_future_doors:door2a 1    my_future_doors:door7a 1
same recipe my_future_doors:door9a 1    my_future_doors:door7a 1
same recipe my_future_doors:door8a 1    my_future_doors:door7a 1
same recipe my_future_doors:door6a 1    my_future_doors:door4a 1
same recipe my_future_doors:door2a 1    my_future_doors:door4a 1
same recipe my_future_doors:door9a 1    my_future_doors:door4a 1
same recipe my_future_doors:door8a 1    my_future_doors:door4a 1
same recipe my_future_doors:door7a 1    my_future_doors:door4a 1
same recipe my_future_doors:door6a 1    my_future_doors:door3a 1
same recipe my_future_doors:door2a 1    my_future_doors:door3a 1
same recipe my_future_doors:door9a 1    my_future_doors:door3a 1
same recipe my_future_doors:door8a 1    my_future_doors:door3a 1
same recipe my_future_doors:door7a 1    my_future_doors:door3a 1
same recipe my_future_doors:door4a 1    my_future_doors:door3a 1
same recipe my_future_doors:door6a 1    my_misc_doors:door2a 1
same recipe my_future_doors:door2a 1    my_misc_doors:door2a 1
same recipe my_future_doors:door9a 1    my_misc_doors:door2a 1
same recipe my_future_doors:door8a 1    my_misc_doors:door2a 1
same recipe my_future_doors:door7a 1    my_misc_doors:door2a 1
same recipe my_future_doors:door4a 1    my_misc_doors:door2a 1
same recipe my_future_doors:door3a 1    my_misc_doors:door2a 1
```

Please check the items and modify the recipe of one of them to get the modpack full usable in survival

PS: great work! I cannot build so many buildings to use all the doors ;)

**Comment by kakalak-lumberJack** [comment]
Made a PR to address this issue. There are now unique crafts for each door. My fork also fixes items not being removed from ItemStack when placed. According to my own preference I also made the crafts produce 2 doors, since they will virtually always be crafted in pairs and two steel blocks in the craft is fairly expensive. Probably should have made separate PR's for these changes, but I made the changes on the fly. Apologies.
